tests/foreman/ui/test_host.py

```diff
@@ -2234,6 +2234,69 @@ def test_positive_read_details_page_from_new_ui(
         assert values['Overview']['DetailsCard']['details']['comment'] == 'Host with fake data'
 
 
+@pytest.mark.tier4
+@pytest.mark.stubbed
+def test_positive_update_delete_package():
+    """Update package on host using the new Content tab
+
+    :id: ffc19a40-85f4-4894-a18b-f6d88b2ce377
+
+    :steps:
+        1. Install a package on registered host.
+        2. Navigate to Content tab.
+        3. Check is package is in upgradable state.
+        4. Select package and upgrade via rex.
```

Comment:
I believe the traces tab can be tested within this test, as you updated the package, (if it is a service it needs to be restarted once its updated, but that would add another level to this test, so its better to think about it from the start, or we can create a separate test, let me know what you think )